### PR TITLE
WTrackProperty: double-clicked property field is focused in DlgTrackInfo

### PR DESCRIPTION
**Pull request by ronso0**
A little helper for tag editing:
* double-click [property] label to open Track Info
* if there is a line edit for [property] it is focused, start typing

### CODE REVIEW

File: src/library/dlgtrackinfo.cpp [hunk 1]
```diff
@@ -484,6 +500,16 @@ void DlgTrackInfo::loadTrack(const QModelIndex& index) {
     }
 }
 
+void DlgTrackInfo::focusField(const QString& property) {
+    if (property.isNull() || property.isEmpty()) {
```
Comment (daschuer) [inline src/library/dlgtrackinfo.cpp:504]: isEmpty() includes isNull()
```suggestion
    if (property.isEmpty()) {
```

File: src/library/dlgtrackinfo.h [hunk 2]
```diff
@@ -121,6 +123,8 @@ class DlgTrackInfo : public QDialog, public Ui::DlgTrackInfo {
     TapFilter m_tapFilter;
     mixxx::Bpm m_lastTapedBpm;
 
+    QMultiHash<QString, QWidget*> m_propertyWidgets;
```
Comment (daschuer) [inline src/library/dlgtrackinfo.h:126]: > A multi-valued hash is a hash that allows multiple values with the same key.
Is this the case here? 

```suggestion
    QHash<QString, QWidget*> m_propertyWidgets;
```

**Review by daschuer — CHANGES_REQUESTED** [review]
The trick works here, left some comments.

File: src/library/dlgtrackinfo.h (hunk 2 above)
Comment (ronso0) [reply src/library/dlgtrackinfo.h]: Nope, just dumb copy/pasting.
fixed

**Review by daschuer — APPROVED** [review]
Than you.

**Comment by ronso0** [comment]
**TODO**
document double-click in the manual

**Pull request merged**